easybuild/easyconfigs/a/Artemis/Artemis-16.0.18-Java-1.8.0_152.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'JAR'
+
+name = 'Artemis'
+version = '16.0.18'
+
+homepage = 'http://www.sanger.ac.uk/science/tools/artemis'
+description = """Artemis is a free genome browser and annotation tool that allows visualisation of sequence features,
+ next generation data and the results of analyses within the context of the sequence, and also its six-frame
+ translation."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['ftp://ftp.sanger.ac.uk/pub/resources/software/artemis/v16/v%(version)s']
+sources = ['%(namelower)s-v%(version)s.jar']
+
+java = 'Java'
+javaver = '1.8.0_152'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [
+    (java, javaver),
+    ('X11', '20160819', '-foss-2015b'),
+]
+
+# add the installation dir to PATH
+modextrapaths = {
+    'PATH': '',
+}
```

Comment:
no need to wrap this across multiple lines (and the comment is maybe a bit too much?)

```python
modextrapaths = {'PATH': ''}
```